packages/nocodb/src/cache/NocoCache.ts

```
@@ -129,6 +129,71 @@ export default class NocoCache {
     return this.client.update(`${this.prefix}:${key}`, updateObj);
   }
 
+  public static async setHash(
+    key: string,
+    hash: Record<string, any>,
+    options: {
+      ttl?: number;
+    } = {},
+  ): Promise<boolean> {
+    if (this.cacheDisabled) return Promise.resolve(true);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)